js/main.js

```diff
@@ -0,0 +1,94 @@
+const goods = [
+    {
+      "id": 1,
+      "name": "Mango People T-shirt",
+      "price": 52,
+      "category": "men",
+      "type": "shirt",
+      "size": [
+        "S",
+        "M"
+      ],
+      "icon": "img/featured1.jpg"
+    },
+    {
+      "id": 2,
+      "name": "Mango People T-shirt",
+      "price": 48,
+      "category": "women",
+      "type": "shoes",
+      "size": [
+        "XXS",
+        "XS",
+        "S",
+        "M",
+        "L"
+      ],
+      "icon": "img/featured2.jpg"
+    },
+    {
+      "id": 3,
+      "name": "Mango People T-shirt",
+      "price": 29,
+      "category": "kids",
+      "type": "shirt",
+      "size": [
+        "XS",
+        "S",
+        "M",
+        "L"
+      ],
+      "icon": "img/featured3.jpg"
+    },
+    {
+      "id": 4,
+      "name": "Mango People T-shirt",
+      "price": 46,
+      "category": "women",
+      "type": "shoes",
+      "size": [
+        "XS"
+      ],
+      "icon": "img/featured4.jpg"
+    },
+    {
+      "id": 5,
+      "name": "Mango People T-shirt",
+      "price": 30,
+      "category": "men",
+      "type": "shoes",
+      "size": [
+        "XS",
+        "S",
+        "L",
+        "XL",
+        "XXL"
+      ],
+      "icon": "img/featured5.jpg"
+    }
+]
+
+const renderGoodsItem = (name, price, icon) => 
+     `<div class="production-item">
+            <a href="single.html" class="production-item-link">
+                <img src="${icon}" alt="" class="production-item-img">
+                <div class="production-item-text">
+                    <p class="production-item-title">${name}</p>
+                    <p class="production-item-price">$${price}.00</p>
+                </div>
+            </a>
+            <div class="production-item-addtocart">
+                <a href="" class="production-addtocart-link">
+                    <img src="img/icons/cart_white.svg" alt="" class="production-cart-icon">
+                    <span class="production-addtocart-text">Add to Cart</span>
+                </a>
+            </div>
+        </div>`;
+
+  const renderGoodsList = (list) => {
+    let goodsList = list.map(item => renderGoodsItem(item.name, item.price, item.icon)).join('');
```

Comment:
переменная не меняется, поэтому лучше const
```suggestion
    const goodsList = list.map(item => renderGoodsItem(item.name, item.price, item.icon)).join('');
```